src/services/s3.js

```
@@ -107,6 +107,29 @@ class S3 {
         }
     }
 
+    /**
+     * List files inside the Bucket
+     *
+     * @param  {string} bucketName - the bucket name
+     * @param  {object} params     - additional parameters
+     * @return {Object}            - An object containing the list of files inside the Bucket.
+     */
+
+    async listBucket(bucketName, params) {
+
+        try {
+            const opts = Object.assign(
+                {},
+                { Bucket: bucketName },
+                params
+            );
+            return (await $._s3.listObjectsV2(opts).promise()).Contents;
+        }
+        catch (err) {
```

Don't `try/catch` anything if nothing is being doing with the `err` in the `catch()`.  Just let the error bubble up naturally.